package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
`ESLint` has a lot of runtime dependencies like `espree` or `escope` (and more). Some of those dependencies might be written in ES6 and compiled to ES5 only when published to npm - that's why submoduling doesn't make much sense here. Only thing that comes to my mind is creating a repo for installed ESLint and submoduling that, but this is quite weird. What's the problem with npm?